docs/getting-started.md

```diff
@@ -0,0 +1,325 @@
+# Getting started with JupyterHub
+
+This document describes some of the basics of configuring JupyterHub to do what you want.
+JupyterHub is highly customizable, so there's a lot to cover.
+
+
+## Installation
+
+See [the readme](../README.md) for help installing JupyterHub.
+
+
+## JupyterHub's default behavior
+
+Let's start by describing what happens when you type `sudo jupyterhub`
+after installing it, without any configuration.
+
+
+### Authentication
+
+The default Authenticator that ships with JupyterHub
+authenticates users with their system name and password (via [PAM][]).
+Any user on the system with a password will be allowed to start a notebook server.
+
+
+### Spawning servers
+
+The default Spawner starts servers locally as each user,
+one for each server. These servers listen on localhost,
+and start in the given user's home directory.
+
+
+### Network
+
+JupterHub consists of three main categories of processes:
+
+- Proxy
+- Hub
+- Spawners
+
+The Proxy is the public face of the service.
+Users access the server via the proxy.
+By default, this is listening on all public interfaces on port 8000.
+You can access the hub at:
```

Comment:
I mean JupyterHub. I don't want to describe it as the proxy, because then people think they want the hub, and skip to using only the hub's private address, which doesn't work (this has actually happened).